### ISSUE TITLE
PostgreSQL 16 pg_dump not found in make catalog backup script

### ISSUE THREAD

**Issue by J-roen** [bacularis/bacularis-docker#7]
Since I upgraded Bacula(ris) to the latest version, my backup-catalog job started to fail with the following error:

```
build-3-18-x86_64-dir JobId 36342: shell command: run BeforeJob "/etc/bacula/scripts/make_catalog_backup bacula"
build-3-18-x86_64-dir JobId 36342: BeforeJob: /etc/bacula/scripts/make_catalog_backup: exec: line 92: /usr/libexec/postgresql16/pg_dump: not found
build-3-18-x86_64-dir JobId 36342: Error: Runscript: BeforeJob returned non-zero status=127. ERR=Child exited with code 127
```

It turns out that the Bacularis Docker image installs PostgreSQL 15, while `/etc/bacula/scripts/make_catalog_backup` contains `BINDIR=/usr/libexec/postgresql16`.

Is this caused by a mismatch between Bacula and Bacularis, or by an error in my own configuration?

I worked around it by installing PostgreSQL 16 in my Docker container: `apk add postgresql16`

Thanks in advance!

**Comment by ganiuszka** [comment]
Hello @J-roen

Thanks for report. Your research of this problem is fully correct.

At the moment I committed fix for that. New container images will be built when we do a new Bacularis release.

We will also add a information to the release notes that PostgreSQL in the ``bacularis-standalone`` and ``bacularis-api-dir`` images has been updated to version ``16`` and it will require doing upgrade between major PostgreSQL versions ``15`` -> ``16`` by ``pg_upgrade``.

Thanks!

Best regards,
Marcin Haba (gani)
